test/github-issues/3357/issue-3357.ts

```diff
@@ -0,0 +1,78 @@
+import "reflect-metadata"
+import { expect } from "chai"
+import { DataSource } from "../../../src/data-source/DataSource"
+import {
+    createTestingConnections,
+    closeTestingConnections,
+} from "../../utils/test-utils"
+import { BugInitial } from "./entity/BugInitial"
+import { BugUpdated } from "./entity/BugUpdated"
+
+describe("github issues > #3357 (postgres) changing varchar length should ALTER, not DROP/ADD", () => {
```

Comment:
We are moving away from creating tests for specific GitHub issues.

I think the tests for this feature could be added to [test/functional/schema-builder/change-column.ts](https://github.com/typeorm/typeorm/blob/master/test/functional/schema-builder/change-column.ts) instead.